specification/search/resource-manager/Microsoft.Search/preview/2019-10-01-preview/search.json

```diff
@@ -777,6 +828,272 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Search/searchServices/{searchServiceName}/privateEndpointConnections": {
+      "get": {
+        "tags": [
+          "PrivateEndpointConnections"
+        ],
+        "x-ms-pageable": {
+          "nextLinkName": null
```

Comment:
>null [](start = 26, length = 4)

"nextLink"

just adding paging support in swagger.. otherwise, adding support later requires new api version

Reply:
consider updating existing definitions as well

---
In reply to: [389217784](https://github.com/Azure/azure-rest-api-specs/pull/8572#discussion_r389217784) [](ancestors = 389217784)

Reply:
esp in new api version

---
In reply to: [389217804](https://github.com/Azure/azure-rest-api-specs/pull/8572#discussion_r389217804) [](ancestors = 389217804,389217784)

Reply:
Fixed in next commit